### ISSUE TITLE
[FEATURE] UI for the adding questions while creating the test and adding questions can be definitely improved a lot. The radio button is not properly positiioned nor is the remove button. The color scheme does not compliment the overall website make it stand out and look very different from overall website.

### ISSUE THREAD

**Issue by om9090** [vishalverma9572/QuizMaster#79]
### Is this a unique feature?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Is your feature request related to a problem/unavailable functionality? Please describe.

Well, as you can see the color scheme doesnt match properly with the whole website also the placement of Input components in the box like option, correct, remove button is not proper. It can be improved to be visually more appealing.
![Screenshot 2024-10-06 042713](https://github.com/user-attachments/assets/b79f8444-a44e-4662-9dcf-794a13da500e)

### Proposed Solution

I can fix the alignmnent of component along with color scheme making it more visually appealing and interactive to the user to ensure it doesn't face any difficulty.

### Screenshots

before:
![Screenshot 2024-10-06 042713](https://github.com/user-attachments/assets/f0d4938d-98aa-41a7-9879-a2a372d47fa5)
after:
![Screenshot 2024-10-06 043424](https://github.com/user-attachments/assets/87943ee3-b750-4a35-afa5-3d1edba8b444)

### Do you want to work on this issue?

Yes

### If "yes" to above, please explain how you would technically implement this (issue will not be assigned if this is skipped)

I'm a frontend developer for more than 1.5 years and I can definitely make good UI for interactive websites. Moreover, making this feature in component will allow you to use in anywhere in website just passing few parameters. It can thus help in code reusability.
Can you please assign this issue to under hacktoberfest?
I won't dissappoint you @vishalverma9572

**Comment by vishalverma9572** [comment]
@om9090 , Definitely, we can improve the UI of our "Create Test" page, and you should focus on that. However, since our website currently has a limited database and only supports text-based questions and options, please ensure that the UI improvements align with this limitation.

**Comment by om9090** [comment]
Thank you. I’ll start working on it.
